server/src/main/java/org/elasticsearch/index/search/stats/ShardSearchStats.java

```diff
@@ -26,9 +27,15 @@
 
 public final class ShardSearchStats implements SearchOperationListener {
```

Comment:
I believe the `SearchOperationListener` only covers the search activity that we receive via `_search`. 
Do we need to check monitor the `_query` activity as well? (i.e. ES|QL) What's the plan here?